scss/atoms/inputs/_inputs.scss

```diff
@@ -0,0 +1,31 @@
+/*
+* ==========================================================================
+* Inputs
+* ==========================================================================
+*/
+
+@import
+	"dist/chassis",
+	"mixins";
+
+input[type=text],
```

Comment:
They're coming. I will keep updating this PR just like the other two I have opened until all is done. Like @geekman-rohit suggested to open a PR so that the code can be reviewed early on as I go.